### PR TITLE
cb-85 Update the tags based on inubekit.

### PR DESCRIPTION
**Pull request by Angelbt12**
tag settings in requirements
image

### CODE REVIEW

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]
the component still needs to be updated in some files

image

**Comment by novadeveloper14** [comment]
conflicts need to be resolved

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]
need to update in:

src\components\data\TableBoard\stories\mockStories.tsx
src\pages\board\outlets\financialReporting\PromissoryNotes\config.tsx

File: src/components/data/TableBoard/stories/mockStories.tsx [hunk 1]
```diff
@@ -6,8 +6,9 @@ import {
   MdOutlineCheckCircle,
   MdRemove,
 } from "react-icons/md";
-import { Icon, Stack, Tag } from "@inube/design-system";
+import { Icon, Stack} from "@inube/design-system";
 
+import {Tag} from "@inubekit/tag";
```
Comment (novadeveloper14) [inline src/components/data/TableBoard/stories/mockStories.tsx:11]: This is an external export of the design system library, adjust import format

File: src/pages/board/outlets/financialReporting/PromissoryNotes/config.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ import {
 } from "react-icons/md";
 import { Icon } from "@inubekit/icon";
 
-import { Tag } from "@components/data/Tag";
+import { Tag } from "@inubekit/tag";
```
Comment (novadeveloper14) [inline src/pages/board/outlets/financialReporting/PromissoryNotes/config.tsx:11]: This is an external export of the design system library, adjust import format

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]

File: src/components/data/TableBoard/stories/mockStories.tsx (hunk 1 above)
Comment (Angelbt12) [reply src/components/data/TableBoard/stories/mockStories.tsx:11]: ok

File: src/pages/board/outlets/financialReporting/PromissoryNotes/config.tsx (hunk 2 above)
Comment (Angelbt12) [reply src/pages/board/outlets/financialReporting/PromissoryNotes/config.tsx:11]: ok

File: src/components/data/TableBoard/stories/mockStories.tsx (hunk 1 above)
Comment (Angelbt12) [reply src/components/data/TableBoard/stories/mockStories.tsx:11]: ok

File: src/components/data/TableBoard/stories/mockStories.tsx [hunk 3]
```diff
@@ -6,7 +6,9 @@ import {
   MdOutlineCheckCircle,
   MdRemove,
 } from "react-icons/md";
-import { Icon, Stack, Tag } from "@inube/design-system";
+
+import { Icon, Stack} from "@inube/design-system";
```
Comment (novadeveloper14) [inline src/components/data/TableBoard/stories/mockStories.tsx:10]: import { Icon, Stack} from "@inube/design-system";
import {Tag} from "@inubekit/tag";

These two exports are external so they should be at the top of the imports, it is not an internal import with aliases, it is an export of a module.

File: src/pages/board/outlets/financialReporting/PromissoryNotes/config.tsx [hunk 4]
```diff
@@ -6,9 +6,10 @@ import {
   MdRemove,
   MdClose,
 } from "react-icons/md";
+
 import { Icon } from "@inubekit/icon";
+import { Tag } from "@inubekit/tag";
```
Comment (novadeveloper14) [inline src/pages/board/outlets/financialReporting/PromissoryNotes/config.tsx:11]: import { Icon } from "@inubekit/icon";
import { Tag } from "@inubekit/tag";

These two exports are external so they should be at the top of the imports, it is not an internal import with aliases, it is an export of a module.

**Review by novadeveloper14 — CHANGES_REQUESTED** [review]

File: src/components/data/TableBoard/stories/mockStories.tsx (hunk 3 above)
Comment (Angelbt12) [reply src/components/data/TableBoard/stories/mockStories.tsx:10]: ok

File: src/pages/board/outlets/financialReporting/PromissoryNotes/config.tsx (hunk 4 above)
Comment (Angelbt12) [reply src/pages/board/outlets/financialReporting/PromissoryNotes/config.tsx:11]: ok

**Review by novadeveloper14 — APPROVED** [review]

**Review by carin91 — APPROVED** [review]

**Pull request merged**